charts/neon-proxy/values.yaml

```
@@ -131,6 +131,26 @@ settings:
   # settings.extraCmdFlags -- (list) additional arguments to proxy binary
   extraCmdFlags: []
 
+pgSniRouter:
+  # pgSniRouter.destination -- append this domain zone to the transformed SNI hostname to get the destination address, e.g. "svc.cluster.local"
+  destination: "svc.cluster.local"
+  # pgSniRouter.domain -- domain used in TLS cert for client postgres connections
+  domain: "dummy"
+
+  exposedService:
+    # pgSniRouter.exposedService.annotations -- Annotations to add to the exposed service
+    annotations: {}
+      # service.beta.kubernetes.io/aws-load-balancer-type: external
+      # service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: ip
+      # service.beta.kubernetes.io/aws-load-balancer-scheme: internet-facing
+    # external-dns.alpha.kubernetes.io/hostname: chart-example.local
```

This is an example for an annotation, right?
```suggestion
      # external-dns.alpha.kubernetes.io/hostname: chart-example.local
```